### PR TITLE
quartermasters get their voice of god power boost correctly

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
we should really just have an easy way to set up new head jobs instead of manually doing the same shit for every single one

## Changelog
:cl:
fix: quartermasters get their voice of god power boost correctly
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/quartermaster.dm [hunk 1]
```diff
@@ -34,6 +34,7 @@
 	)
 	rpg_title = "Steward"
 	job_flags = STATION_JOB_FLAGS | JOB_BOLD_SELECT_TEXT | JOB_CANNOT_OPEN_SLOTS
+	voice_of_god_power = 1.4 //Command staff has authority
```
Comment (LemonInTheDark) [inline code/modules/jobs/job_types/quartermaster.dm:37]: this is horrible

**Review by LemonInTheDark — APPROVED** [review]

**Comment by LemonInTheDark** [comment]
agree

**Comment by MrMelbert** [comment]
```dm
/proc/voice_of_god(mob/god)
   var/modifier = 1
   if(god.mind?.assigned_role.department_bitflag & BITFLAG_COMMAND)
       modifier *= 1.4
```

**Comment by Fikou** [comment]
> ```dm
> /proc/voice_of_god(mob/god)
>    var/modifier = 1
>    if(god.mind?.assigned_role.department_bitflag & BITFLAG_COMMAND)
>        modifier *= 1.4
> ```

fr

**Comment by Mothblocks** [comment]
do that then...

**Comment by san7890** [comment]
bro why not

**Comment by LemonInTheDark** [comment]
In scope

**Comment by LemonInTheDark** [comment]
IT WAS IN SCOPE

**Comment by Fikou** [comment]
NUH UH

**Pull request merged**